Mathlib/Data/Nat/Choose/Sum.lean

```diff
@@ -235,4 +235,11 @@ theorem sum_antidiagonal_choose_succ_mul (f : ℕ → ℕ → R) (n : ℕ) :
   simpa only [nsmul_eq_mul] using sum_antidiagonal_choose_succ_nsmul f n
 #align finset.sum_antidiagonal_choose_succ_mul Finset.sum_antidiagonal_choose_succ_mul
 
+theorem sum_antidiagonal_choose_add (d n : ℕ) :
+    (Finset.sum (antidiagonal n) fun ij ↦ (Nat.choose (d + ij.2) d)) =
+    Nat.choose (d + n) d + Nat.choose (d + n) (succ d) := by
+  induction' n with n hn
```

Comment:
I think `induction'` will be deprecated sooner than later, can you use `induction` tactic instead